release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -295,6 +295,15 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
 [id="ocp-4-10-scale"]
 === Scale
 
+[id="ocp-4-10-numa-aware-scheduling"]
+==== NUMA-aware scheduling with NUMA Resources Operator
+
+The default {product-title} scheduler does not see individual NUMA zones in the compute node. This can lead to sub-optimal scheduling of latency-sensitive workloads. A new NUMA Resources Operator is available which deploys a NUMA-aware secondary scheduler. The NUMA-aware secondary scheduler makes scheduling decisions for workloads based on a complete picture of available NUMA zones in the cluster. This ensures that latency-sensitive workloads are processed in a single NUMA zone for maximum efficiency and performance.
```

Comment:
```suggestion
The default {product-title} scheduler does not see individual non-uniform memory access (NUMA) zones in the compute node. This can lead to sub-optimal scheduling of latency-sensitive workloads. A new NUMA Resources Operator is available which deploys a NUMA-aware secondary scheduler. The secondary scheduler makes scheduling decisions for workloads based on a complete picture of available NUMA zones in the cluster. This ensures that latency-sensitive workloads are processed in a single NUMA zone for maximum efficiency and performance.
```

Reply:
I think we need to spell out NUMA as an acronym. The other edits are just suggestions based on my ear. Feel free to take what is useful and disregard the rest.